recipes/ofeli/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.40.0"
+
+
+class OfeliConan(ConanFile):
+    name = "ofeli"
+    description = "An Object Finite Element Library"
+    topics = ("ofeli", "finite element", "finite element library",
+              "finite element analysis", "finite element solver")
+    license = "GNU Lesser General Public License (LGPL)."
+    homepage = "http://ofeli.org/index.html"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for Linux")
+        if self.settings.compiler != "gcc":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for GCC")
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        if not self.settings.compiler.cppstd:
+            self._autotools.cppstd_flag = "-std=c++11"
+        self._autotools.configure(args=["--enable-release"])
+        return self._autotools
+
+    def configure(self):
+        if self.settings.os == "Linux":
+            self.settings.compiler.libcxx = "libstdc++11"
```

Comment:
```suggestion
```
Is this really needed ? Is it even allowed to modify the settings from a recipe ? 
`libstdc++` is also compatible with c++11

Reply:
I can try to put it as an option in the configuration, so that the user can choose between libstdc++ and libstdc++11. Is this okay?

Reply:
Why not using the regular way to define settings: `-s` command line argument, or in the profile?

Reply:
i have tried to avoid use libstdc++11 at the beginning but it did not work, so y add it at this commit: https://github.com/conan-io/conan-center-index/pull/8375/commits/ba0b21a1aaa7c4362fa4b301f7ec07cd5e407412, It is possible that the library needs that flag?

Reply:
Hello again, reviewing the history, the main problem is that with the gcc compiler in version 5, we need this configuration to compile correctly.

Reply:
The problem with gcc5 was actually solved in 9ca059b1237f34e693bb344fe85bbf28b5c3d503

Reply:
I just made the changes, I am waiting for the CI to finish. Thanks for your response!

Reply:
Hi again, with the changes you commented, it seems that it fails when executing the test example with gcc 8 :S Any idea why the stdc ++ 11 flag might work?